testdata/jsight_0.3/ser-to-openapi/req.jschema.rules.additionalProperties_03/string.openapi

```diff
@@ -0,0 +1,276 @@
+      "get": {
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "foo": { "type": "string", "example": "bar" }
+                  },
+                  "required": ["foo"]
+                }
+              }
+            }
+          },
+          "201": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "foo": { "type": "string", "example": "bar" }
+                  },
+                  "required": ["foo"],
+                  "additionalProperties": {
+                    "type": "array",
+                    "items": {}
+                  }
+                }
+              }
+            }
+          },
+          "202": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "foo": { "type": "string", "example": "bar" }
+                  },
+                  "required": ["foo"],
+                  "additionalProperties": {
+                    "type": "boolean"
+                  }
+                }
+              }
+            }
+          },
+          "203": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "foo": { "type": "string", "example": "bar" }
+                  },
+                  "required": ["foo"],
+                  "additionalProperties": {
+                    "type": "number"
+                  }
+                }
+              }
+            }
+          },
+          "204": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "foo": { "type": "string", "example": "bar" }
+                  },
+                  "required": ["foo"],
+                  "additionalProperties": {
+                    "type": "integer"
+                  }
+                }
+              }
+            }
+          },
+          "205": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "foo": { "type": "string", "example": "bar" }
+                  },
+                  "required": ["foo"],
+                  "additionalProperties": {
+                    "type": "object"
```

Comment:
Don't we need other fields for object type: `properties` and `additionalProperties`?

Reply:
fixed